app/controllers/users_controller.rb

```diff
@@ -18,10 +21,33 @@ def create
     end
   end
 
+  def edit
+  end
+
+  def update
+    if @user.update_attributes(user_params)
+      flash[:success] = "Profile updated"
+      redirect_to @user
+    else
+      render 'edit'
+    end
+  end
+
   private
 
     def user_params
       params.require(:user).permit(:name, :email, :password,
                                    :password_confirmation)
     end
+
+    # Before actions
+
+    def signed_in_user
+      redirect_to signin_url, notice: "Please sign in." unless signed_in?
```

Comment:
[rubocop] Prefer single-quoted strings when you don't need string interpolation or special symbols.